CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Change Log
 ==========
 
+Version 2.6.0 *(2018-10-25)*
+----------------------------
+- Improved formatting of generated code.
+- Improved logging by putting all debug logging behind the `stagDebug` flag.
+- Improve integration of library by switching to new nullability annotations library.
+- Wrote unit tests for `KnownTypeAdapters`.
+- Fixed bug where code generation was non deterministic by switching to linked versions of `HashSet` and `HashMap`.
+- Added support for turning on/off serialization of `null` with `stag.serializeNulls` compiler option. Default is off, which is a behavior change from version 2.5.1.
+
```

Comment:
Updated build tools. kotlin, and target sdk?

Reply:
Updating tooling and dependency versions have historically not been called out in change logs since they do not affect the end user. As a general rule changes that are not going to affect the end user are normally left out of the change log. Arguably, putting a blurb about writing tests in the change log is similarly unnecessary but I have been doing that so that the progress made on testing is officially communicated to end users.